packages/instant/src/util/gas_price_estimator.ts

```diff
@@ -0,0 +1,44 @@
+import { BigNumber } from '@0x/utils';
+
+import { DEFAULT_GAS_PRICE } from '../constants';
+
+interface GasStationResult {
+    average: number;
+    fastestWait: number;
+    fastWait: number;
+    fast: number;
+    safeLowWait: number;
+    blockNum: number;
+    avgWait: number;
+    block_time: number;
+    speed: number;
+    fastest: number;
+    safeLow: number;
+}
+
+const endpointUrl = 'https://ethgasstation.info/json/ethgasAPI.json';
+const fetchFastAmountInWei = async () => {
+    const res = await fetch(endpointUrl);
+    const gasInfo = (await res.json()) as GasStationResult;
+    const gasPriceInGwei = new BigNumber(gasInfo.fast / 10);
+    return gasPriceInGwei.mul(1000000000);
```

Comment:
This logic should be cleaned up or moved to a util? I'm not really sure what `fast` is denominated in.

Reply:
This was a straight up port from https://github.com/0xProject/0x-monorepo/blob/development/packages/website/ts/blockchain.ts#L941 

Ideally all of this would be moved to a new package and shared between the two, but not sure if that's worth it right now

Reply:
Website code should not be used as the bar lol